_data/packages.yml

```diff
@@ -605,19 +633,21 @@
   issue_link: https://github.com/pyOpenSci/software-submission/issues/115
   joss:
   partners:
-    - numfocus
```

Comment:
Is this correct to be removing this?

Reply:
@sneakers-the-rat good catch. 
we should remove that partner. Our only current valid review partners are pangeo & astropy